packages/domain/src/use-cases/wallets/WalletLogin.ts

```diff
@@ -50,7 +50,7 @@ export class WalletLogin {
     private readonly credentialsIssuer: ICredentialsIssuer,
     private readonly credentialsWriter: ICredentialsWriter,
     private readonly activeWalletPresenter: IActiveWalletPresenter,
-    private readonly connectionErrorPresenter: IConnectionErrorPresenter,
+    private readonly genericPresenter: IWalletLoginPresenter,
```

Comment:
Naming wise, as far as this class know this is a `walletLoginPresenter`.
```suggestion
    private readonly walletLoginPresenter: IWalletLoginPresenter,
```